modules/ROOT/partials/release-notes/6.0-release-notes-summary.adoc

```diff
@@ -276,7 +276,7 @@ The following elements were previously deprecated and have, with this release, b
 
 | `validate`                            | Schema option |
 
-| `XHR`                                 | API           | Any remaining `XHR` users have been replaced with `fetch`.
+| XHR                                   | API           | Any remaining XHR users have been replaced with https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API[`fetch`].
```

Comment:
This is a bit of an old issue, but `users` doesn't look right. Should that be `uses`?

Reply:
Probably. I can’t remember, now, if this is a typo or a mistake copied in from another source. Corrected to _uses_ nonetheless.